test/cli/test_cli_mask.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import pathlib
+import tempfile
+import os
+
+from click.testing import CliRunner
+from shimmingtoolbox.cli.mask import mask_cli
+from shimmingtoolbox import __dir_testing__
+
+
+def test_cli_mask_cube():
+    with tempfile.TemporaryDirectory(prefix='st_' + pathlib.Path(__file__).stem) as tmp:
+        runner = CliRunner()
+
+        inp = os.path.join(__dir_testing__, 'sub-fieldmap', 'fmap', 'sub-fieldmap_phase1.nii.gz')
+        out = os.path.join(tmp, 'nifti1')
+        size = 5
+
+        result = runner.invoke(mask_cli, ['cube', '-input', inp, '-output', out, '-size', size])
+
+        assert result.exit_code == 0
+        assert result is not None
+        assert len(os.listdir(out)) != 0
```

Comment:
it would be nice to check that the expected mask is created

Reply:
I'm going to think about how to add this check

Reply:
You can open it and make sure some pixels are what you are expecting